core/server/src/main/java/alluxio/master/file/FileSystemMasterClientRestServiceHandler.java

```diff
@@ -263,7 +263,9 @@ public Response getStatusInternal(@QueryParam("fileId") Long fileId) {
   /**
    * @summary get the UFS address
    * @return the UFS address
+   * @deprecated since version 1.1 and will be removed in version 2.0
```

Comment:
there is no alternative ... this method should simply not be used, because it allows the client to write code that assuming that there is only one underFS, which is something we want to prevent from happening